packages/debs/SPECS/wazuh-agent/debian/postrm

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+# postrm script for wazuh-agent
+# Wazuh, Inc 2024
+
+set -ex
+export INSTALLATION_WAZUH_DIR="/opt/wazuh-agent"
+WAZUH_TMP_DIR="${INSTALLATION_WAZUH_DIR}/tmp"
+
+case "$1" in
+    remove|failed-upgrade|abort-install|abort-upgrade|disappear)
+
+        if [ -d ${WAZUH_TMP_DIR} ]; then
+            rm -rf ${WAZUH_TMP_DIR}
+        fi
+
+        if [ "$1" = "remove" ]; then
+            rm -f /usr/local/bin/wazuh-agent
+            rm -f /usr/lib/systemd/system/wazuh-agent.service
+            rm -f /usr/local/lib/libdbsync.so
+            rm -f /usr/local/lib/libsysinfo.so
+            rm -f /usr/local/lib/libstdc++.so.6
```

Comment:
Just double checking, but are we sure that it is ok to remove `libstdc++`? Is this one exclusive to the wazuh agent?

Reply:
It is a library that we ‘install’ to the `/usr/local/lib/` folder to run the agent on _CentOS 7_ and _Debian 10_, and by default the `libstdc++` library is located in the path: `/usr/local/gcc-<version>/lib64/libstdc++.so.6`.
> A priori this shouldn't cause any problems, but we can give it another try in case it could be improved.